.cargo/config.toml

```diff
@@ -4,6 +4,7 @@ runner = "espflash flash --monitor"
 [build]
 rustflags = [
   "-C", "link-arg=-Tlinkall.x",
+  "-C", "link-arg=-Trom_functions.x",
```

Comment:
This will cause linker errors when not using `esp-wifi` (as you can see by the failing CI)